stable/grafana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 3.10.2
+version: 3.10.3
```

Comment:
Need to bump chart version to the latest again.

Reply:
@zanhsieh done!